src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MODULE_COMMANDS_NAME_MAP } from 'src/modules/transaction/configuration/moduleCommand';
+import BoxHeader from 'src/theme/box/header';
+import BoxContent from 'src/theme/box/content';
+import { Input } from 'src/theme';
+import TxComposer from '@transaction/components/TxComposer';
+import { convertCommissionToNumber } from '@pos/validator/utils';
+import { useCurrentCommissionPercentage } from '@pos/validator/hooks/useCurrentCommissionPercentage';
+import InputLabel from './InputLabel';
+import styles from './ChangeCommissionForm.css';
+
+export const ChangeCommissionForm = ({ nextStep }) => {
+  const { t } = useTranslation();
+  const { currentCommission, isLoading, isSuccess: isCommissionSuccess } = useCurrentCommissionPercentage();
+  const [newCommission, setNewCommission] = useState(currentCommission);
+
+  useEffect(() => {
+    if (currentCommission && currentCommission !== newCommission) {
+      setNewCommission(currentCommission);
+    }
+  }, [isCommissionSuccess]);
+
+  const onConfirm = (formProps, transactionJSON, selectedPriority, fees) => {
+    nextStep({
+      selectedPriority,
+      formProps,
+      transactionJSON,
+      fees,
+    });
+  };
+
+  const newCommissionParam = convertCommissionToNumber(newCommission)
```

Comment:
Should probably do something like 
```
  const newCommissionParam = newCommission && convertCommissionToNumber(newCommission)
  const isFormValid = newCommissionParam && newCommissionParam >= 0 && newCommissionParam <= 10000
  ```
  To prevent empty string from being a valid form value 
  
<img width="200" alt="image" src="https://user-images.githubusercontent.com/8784876/218074410-4c9bacb7-0c9a-4345-af64-48d201879e33.png">